### DISCUSSION TITLE
比心云（Vortex）+中转的API Key如何配置

### DISCUSSION THREAD

**Discussion by GuoPT1** [binary-husky/gpt_academic#2002] [Q&A]
您好，求助大神，我用的是比心云（Vortex）的魔法，API用的是从小爱AI购买的中转的API Key，配置的时候总是显示代理服务器连接超时。我应该怎么办？

**Comment by doctorgreen4721** [comment]
首先确保代理配置没有问题，换一个节点试试。
其次如果是中转api的话其实可以直连吧？
